cljfmt/src/cljfmt/core.cljc

```diff
@@ -41,16 +41,16 @@
   (nil? (z/up* zloc)))
 
 (defn- top? [zloc]
-  (and zloc (not= (z/node zloc) (z/root zloc))))
+  (some-> zloc z/up root?))
 
-(defn- top [zloc]
+(defn- root [zloc]
```

Comment:
This rename reflects that this helper navigates to the root zloc, which is one level above the top-level zlocs.